src/routing/approved.rs

```diff
@@ -397,12 +411,17 @@ impl Approved {
                 }
             }
             Variant::UserMessage(_) => {
-                if !self.should_handle_user_message(msg.dst()) {
+                // If elder, always handle UserMessage, otherwise handle it only if addressed
```

Comment:
Any particular reason to carry out this change (and the same two below) ?
I'd prefer using the previous one to have check and comment in a separate function.

Reply:
I thought that as those functions are all one-liners and each used only once, there is not much reason having them. But I can revert if you thing it's cleaner.

Reply:
I'd prefer revert them, as the comments here is making this already lengthy function expanded further.